deno/payloads/v10/_interactions/_applicationCommands/permissions.ts

```diff
@@ -40,10 +40,16 @@ export interface APIApplicationCommandPermission {
 	permission: boolean;
 }
 
+export enum ApplicationCommandPermissionsConstant {
+	Everyone = 'guild_id',
```

Comment:
This is not useful for users at all. Lets make this an object where we take in a bigint|string, and return the expected value back  (also no, you don't pass in the literal `guild_id` string :D